library/Ivoz/Provider/Domain/Service/BillableCall/CreateOrUpdateByTrunksCdr.php

```diff
@@ -103,6 +103,24 @@ public function execute(
                 ->setEndpointId($trunksCdrDto->getResidentialDeviceId());
         }
 
+        if ($trunksCdrDto->getUserId()) {
+            $billableCallDto
+                ->setEndpointType('User')
```

Comment:
Should EndpointType be a enum? Use Interface::CONSTANT here if true

Reply:
Agree